### ISSUE TITLE
Gflags.exe

### ISSUE THREAD

**Issue by mrexodia** [x64dbg/x64dbg#816]
Implement something like this inside x64dbg.

**Comment by mrexodia** [comment]
Now works correctly, see https://github.com/x64dbg/x64dbg/issues/210#issuecomment-197227768

**Comment by lab313ru** [comment]
Seems doesn't work with chromium-based processes.

**Comment by mrexodia** [comment]
Yes, chromium has a built-in protection against DLL injection and it would also mess with the parent-child relationship, so it wouldn’t work on the sandboxed rendering processes.
